### PR TITLE
bump webpack to 4.21

### PR DESCRIPTION
**Pull request by benelan**
Had to make a change in the webpack config due to a [breaking change](https://github.com/webpack/webpack-dev-server/blob/master/CHANGELOG.md#-breaking-changes-4).

> the contentBase, contentBasePublicPath, serveIndex, staticOptions, watchContentBase, watchOptions were removed in favor of the static option

### CODE REVIEW

**Review by andygup — CHANGES_REQUESTED** [review]
Good catch on the `webpack-dev-server` breaking change, I knew that was coming. It might be a good idea to doc that in the webpack README since it has a good chance of tripping up folks. Maybe just a note at the top of the README similar to what we've done in the Angular repo...

**Comment by benelan** [comment]
> It might be a good idea to doc that in the webpack README since it has a good chance of tripping up folks.

Doc'd it!

**Review by andygup — APPROVED** [review]
👍

File: esm-samples/webpack/README.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 This repo demonstrates how to use the [`@arcgis/core`](https://www.npmjs.com/package/@arcgis/core) ES modules with webpack.
 
+
+## Known Issues
+- ``webpack-dev-server`` had a [breaking change](https://github.com/webpack/webpack-dev-server/blob/master/CHANGELOG.md#-breaking-changes-4) in ``4.0.0`` which removed ``contentBase`` in favor of the ``static`` option. This sample has been changed accordingly.
```
Comment (andygup) [inline esm-samples/webpack/README.md:7]: @benelan looks like one last small item, can you change the double backticks to single `backticks`?

Comment (benelan) [reply esm-samples/webpack/README.md]: Good to go!

Comment (andygup) [reply esm-samples/webpack/README.md]: Awesome, merging!

**Pull request merged**